### PR TITLE
completed-header

### PR DESCRIPTION
**Pull request by Rehmatulla-Khan**

### CODE REVIEW

**Review by faran4engg — COMMENTED** [review]
Add screenshot or gif of the working app

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./_header.scss";
```
Comment (faran4engg) [inline src/components/header/Header.jsx]: This import should come last

File: src/components/header/_header.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  }
+
+  &__icons {
+    display: flex;
+    flex: 0.15;
+    justify-content: space-around;
+    align-items: center;
+
+    img {
+      margin-left: 5px;
```
Comment (faran4engg) [inline src/components/header/_header.scss]: Can this be 0.5 rem?

File: src/components/header/_header.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
+@import "../../_base.scss";
+
+.header {
```
Comment (faran4engg) [inline src/components/header/_header.scss:3]: Where is this div with header class

Comment (Rehmatulla-Khan) [reply src/components/header/_header.scss:3]: @faran4engg  its a Wrapper in Header component

File: src/components/header/_header.scss (hunk 2 above)
Comment (Rehmatulla-Khan) [reply src/components/header/_header.scss]: @faran4engg done

File: src/components/header/Header.jsx (hunk 1 above)
Comment (Rehmatulla-Khan) [reply src/components/header/Header.jsx]: @faran4engg  done

**Pull request merged**